src/NServiceBus.Core/Hosting/HostingComponent.cs

```diff
@@ -12,14 +12,14 @@
 
     partial class HostingComponent
```

Comment:
we might be able to complete remove the Hosting component and leave just the hosting settings around to make the role of the hosting component in regards to the actual hosting logic less confusing.